.github/workflows/reusable-build-and-push.yml

```diff
@@ -12,63 +12,132 @@ on:
         type: boolean
         description: 'If set to true, push the image.'
         default: false
+      stable_release:
+        required: false
+        type: boolean
+        description: 'If set to true the image is pushed to the stable repository otherwise pushed to the development.'
+        default: false
```

Comment:
this might be a little confusing because main merges should go into the stable repo but they aren't what I would call "stable" releases. Perhaps release repository vs development would be a better differentiator?

Reply:
Valid concern. Renamed the parameter to make it less confusion.